README.md

```diff
@@ -23,15 +23,18 @@ Amazon Simple Notification Service (SNS) to send SMS.
    * WEB_SERVICES R
    * ADMIN_NOTIF_QUEUE RE
    * ADMIN_USERS R
-   *  ADMIN_INTEGRATION_LOG RA
+   * ADMIN_INTEGRATION_LOG RA
+   
+   The following privs are required if phoneNumberField is used:
    * \<User Trackor Type\> R
    * \<User Trackor Type Tab containing phoneNumberField\> R
+
 4. Fill the integration settings file:
    - oneVizionUrl - OneVizion URL
    - oneVizionLogin - OneVizion Login
    - oneVizionPwd - OneVizion Password
    - serviceId - ID of the Notification Service created at step 2
-   - phoneNumberField - The name of the field which contains the phone number of the recipient. Recipient is a user trackor related with User ID from the Notif Queue record.
+   - phoneNumberField (Optional. Needed when there are no phones numbers in User Settings) - The name of the field which contains the phone number of the recipient. Recipient is a user trackor related with User ID from the Notif Queue record.
```

Comment:
(Optional. When phone number from the Field should be used instead of one specified in the User Settings)

Reply:
Fixed